src/index.js

```diff
@@ -14,7 +15,9 @@ const root = document.getElementById('root');
 ReactDOM.render(
   <React.StrictMode>
     <ApolloProvider client={client}>
-      <App />
```

Comment:
I think you should put the strict mode back

Reply:
Isn't strict mode there? At least on my screen it is lol

Reply:
did I tell you I'm blind -_- ? 
Yeah, that's great